pkg/services/sqldb/bind.go

```diff
@@ -16,6 +16,15 @@ func (s *serviceManager) ValidateBindingParameters(
 	return nil
 }
 
+//TODO: What behavior do we want for bind on a non-bindable service.
+//Appropriate error?
+func (s *serverOnlyServiceManager) Bind(
+	instance service.Instance,
+	_ service.BindingParameters,
+) (service.BindingContext, service.Credentials, error) {
+	return nil, nil, nil
+}
+
```

Comment:
This is fine for now-- we'll just no-op it. We can figure out how to deal with this more elegantly later.